<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrugsWidget::Internal::DrugsCentralWidget</class>
 <widget class="QWidget" name="DrugsWidget::Internal::DrugsCentralWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="DrugSelector" name="m_DrugSelector" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="PrescriptionViewer" name="m_PrescriptionView" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PrescriptionViewer</class>
   <extends>QWidget</extends>
   <header location="global">drugsplugin/drugswidget/prescriptionviewer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DrugSelector</class>
   <extends>QWidget</extends>
   <header location="global">drugsplugin/drugswidget/drugselector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
